Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    structure = np.dstack([structure_spec, structure, structure_spec])
</pre><p>Realized I had a bug here. To fix it:</p>

<pre class="ql-syntax" spellcheck="false">structure = np.swapaxes(np.dstack([structure_spec, structure, structure_spec]), 0, 2)
</pre><p>so the spectral axis is the 0th, not the 2nd the <code>dstack</code> creates </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>